### ISSUE TITLE
ng build error 0.0.29 ENOENT: no such file or directory, lstat 'public'

### ISSUE THREAD

**Issue by guillaumeleone** [angular/angular-cli#386]
Hi team,

Info: 
```
$ node --version
v5.0.0
$ npm --version
3.3.6
```

We have an error with the new version of angular-cli.

```
/workspace
└── angular-cli@0.0.29 

Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://www.ember-cli.com/user-guide/#watchman for more info.
BuildingBuild failed.
The Broccoli Plugin: [Funnel] failed with:
Error: ENOENT: no such file or directory, lstat 'public'
```
I (re)try with angular-cli@0.0.28 it works.

Do you have any idea ?

**Comment by anode7** [comment]
Hi @guillaumeleone 
test run " ng init " in your project ... by answering "no" to the Overwrite of the files of your "src" project

**Comment by anode7** [comment]
Why create the folder "/public" in angular-cli@0.0.29?

**Comment by anode7** [comment]
ok #380

**Comment by hansl** [comment]
Fixed as of PR #380
